src/scripts/templates/customers/addresses.js

```diff
@@ -7,56 +7,54 @@
  * @namespace customerAddresses
  */
 
-import $ from 'jquery';
-
-const $newAddressForm = $('#AddressNewForm');
-
-if ($newAddressForm.length) {
-  // Initialize observers on address selectors, defined in shopify_common.js
-  if (Shopify) {
-    new Shopify.CountryProvinceSelector(
-      'AddressCountryNew',
-      'AddressProvinceNew',
-      {
-        hideElement: 'AddressProvinceContainerNew',
-      },
-    );
-  }
-
-  // Initialize each edit form's country/province selector
-  $('.address-country-option').each(function() {
-    const formId = $(this).data('form-id');
-    const countrySelector = `AddressCountry_${formId}`;
-    const provinceSelector = `AddressProvince_${formId}`;
-    const containerSelector = `AddressProvinceContainer_${formId}`;
+import {CountryProvinceSelector} from '@shopify/theme-addresses';
+
+const selectors = {
+  addressContainer: '.address-wrapper',
```

Comment:
For separation of concerns, we try to avoid using any class selectors and solely rely on data-selector as JS selectors now. Can you fix the couples of one using class selectors?